### PR TITLE
Add comparison

### PR DESCRIPTION
**Pull request by abramov-oleg**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -158,3 +158,22 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Savant downloads and models
+# ignore everything except labels.txt in a subdirectory
```
Comment (dorgun) [inline .gitignore]: Why did you exclude label.txt? If you need label.txt, you can create it in the onnx export step. In ultralytics, the YOLO class has a names property that returns class names.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+build-pytorch:
+	docker build -t $(PYTORCH_IMAGE_NAME) docker/pytorch
+
+pull-savant:
+	docker pull $(SAVANT_IMAGE_NAME)
+
+run-savant:
+	docker run --rm --gpus=all \
+		-e MODEL_PATH=/cache/models/$(SAVANT_MODULE_NAME) \
+		-v $(shell pwd)/src/savant:/opt/savant/samples/$(SAVANT_MODULE_NAME) \
```
Comment (dorgun) [inline Makefile]:
```suggestion
		-v `pwd`/src/savant:/opt/savant/samples/$(SAVANT_MODULE_NAME) \
```

File: src/pytorch/optimized_pytorch_decode.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    :return: dict with crop+resize decoder options
+    """
+    video_height, video_width = video_size
+    infer_height, infer_width = infer_size
+
+    old_aspect = video_width / video_height
+    new_aspect = infer_width / infer_height
+
+    if old_aspect > new_aspect:
+        # Crop width
```
Comment (dorgun) [inline src/pytorch/optimized_pytorch_decode.py:69]: Why is the original frame cropped?

**Review by dorgun — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (bwsw) [reply Makefile]: Why not `$(pwd)`?

Comment (abramov-oleg) [reply Makefile]: > Why not `$(pwd)`?

This won't work as this is Makefile, not just shell script. Working alternative would be 

```
$$(pwd)
```

Regarding suggestion, I can change to backticks, no problem, but I don't see anything wrong with the current version.

Comment (abramov-oleg) [reply Makefile]: Changed to backticks.

File: .gitignore (hunk 1 above)
Comment (abramov-oleg) [reply .gitignore]: Reworked ONNX export script, removed `labels.txt` from repo.

**Review by dorgun — APPROVED** [review]

**Pull request merged**